### PR TITLE
better formatting of generator and generated file in orer to increase…

### PR DESCRIPTION
**Pull request by jensens**
… readability of Python code and generated Grundfile.js

### CODE REVIEW

File: Products/CMFPlone/_scripts/_generate_gruntfile.py [hunk 1]
```diff
@@ -134,13 +130,11 @@
                     plugins: [
                         new require('less-plugin-inline-urls'),
                     ],
+                    paths: {less_paths},
                     modifyVars: {{
-                      {globalVars}
+{globalVars}
                     }}
```
Comment (thet) [inline Products/CMFPlone/_scripts/_generate_gruntfile.py]: why this unindent?

**Comment by thet** [comment]
I guess, that's untested code, isn't it?
I'll let run a Pull-Request job anyways: http://jenkins.plone.org/job/pull-request-5.0/988/

**Comment by jensens** [comment]
I dont see any test for it, anyway its also difficult to test. This would need quite a lot of refactoring and out of scope of this PR.

File: Products/CMFPlone/_scripts/_generate_gruntfile.py (hunk 1 above)
Comment (jensens) [inline Products/CMFPlone/_scripts/_generate_gruntfile.py]: because I indent manually and then it fits afterwards (before only first line was indented).

**Pull request merged**